invenio_logging/datastreams/backends.py

```diff
@@ -89,7 +89,7 @@ def search(self, query=None, size=10):
                         "operator": "and",
                     }
                 },
-                "sort": [{"timestamp": {"order": "desc"}}],
+                "sort": [{"@timestamp": {"order": "desc"}}],
```

Comment:
I think this whole method should be in a log dedicated service class 
this should probably not be in this module.
could work as a prototype now, but it should be moved in the next iteration to a service.
Probably even before finalising the frontend, otherwise you will end up patching a lot of things.